ci_framework/roles/switch_config/tasks/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+- name: Load switch configuration
+  ansible.builtin.set_fact:
+    switch_vars: "{{ (lookup('vars', inventory_hostname, default='')) }}"
```

Comment:
What is `inventory_hostname`?

Reply:
The idea is to match the config to what the inventory name of each switch. For example a config for a switch named in the inventory sw01 would look like this and use the vars under it.

`sw01:
  vlans:
    - start: 1
      end: 1
  interfaces:
    - description: "server01"
      iface: "GigabitEthernet4/0/1"
      iface_mode: "access"
      vlan: "1"
`